pxr/usdImaging/usdviewq/primViewItem.py

```diff
@@ -256,6 +260,12 @@ def _nameData(self, role):
                 toolTip = 'Inactive ' + toolTip
             elif self.isInstance:
                 toolTip = 'Instanced ' + toolTip
+        
+            # tooltip should always show both name and display name
+            toolTip = toolTip + "<br>Name: " + self.name
+            if self.displayName != "":
```

Comment:
simply `if displayName:` is simpler and should yield the same results?

Reply:
I tend to prefer explicit checks to implicit ones for the reader but we can change it to `if self.displayName:` if you prefer.

Reply:
I don't know if you'd consider it python programming "style", but my assessment of coding practice here, which I think is what Corey was getting at, is that needing to be aware of whether any particular python function is going to return None or an emoty string or an empty list is a source of fragility, but they all evaluate to `False`, so doing the simple check `if displayName:` leads to more resilient code?  I am not a python expert or style-proselytizer,  but I think this relates to all of the other things you call out below as having encountered...

Reply:
Changed to `if displayName:`